js/background.js

```diff
@@ -290,18 +258,18 @@ function RestoreCollection(collectionIndex, removeCollection)
 	collections = collections.filter(i => i != collections[collectionIndex]);
 	localStorage.setItem("sets", JSON.stringify(collections));
 
-	UpdateTheme();
+	chrome.browserAction.setBadgeText({ text: collections.length < 1 ? "" : collections.length.toString() });
 }
 
 function RemoveTab(collectionIndex, tabIndex)
 {
 	var set = collections[collectionIndex];
-	if (--set.tabsCount < 1)
+	if (set.tabsCount < 2)
```

Comment:
Why don't you decrement the tabsCount anymore ? That means if you have 5 tabs in a collection, and delete one, the `tabsCount` will still be 5 even though one is gone :/ ? Or am I missing an update of that count somewhere ?